tests/cisco6500/test_cisco6500_switch_interface.py

```diff
@@ -21,20 +21,3 @@ class TestCiscoSwitchInterface(ProtocolTest):
     tester_class = SshTester
     test_switch = "cisco6500"
 
-    @with_protocol
```

Comment:
you can leave this test... or delete the file :P

Reply:
Haha! git rm fail...